src/ralph/accounts/management/commands/ldap_sync.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+import logging
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from ldap.controls import SimplePagedResultsControl
+
+
+logger = logging.getLogger(__name__)
+LDAP_RESULTS_PAGE_SIZE = 100
+
+try:
+    import ldap
+    from django_auth_ldap.backend import _LDAPUser
+    ldap_module_exists = True
+except ImportError:
+    logger.debug("Install python-ldap and django_auth_ldap packages")
+    ldap_module_exists = False
+
+
+def _truncate_surname(ldap_dict):
```

Comment:
this could be more generic - we should truncate other fields as well (ex. first name)?